manifests/init.pp

```diff
@@ -69,13 +69,13 @@
 
   if $repo {
     apt::source { 'aptly':
-      location => 'http://repo.aptly.info',
-      release  => 'squeeze',
-      repos    => 'main',
-      key      => {
-        'server' => $key_server,
-        'id'     => '26DA9D8630302E0B86A7A2CBED75B5A4483DA07C',
-      },
+      location   => 'http://repo.aptly.info',
+      release    => 'squeeze',
+      repos      => 'main',
+      key        =>  {
+        key_server => $key_server,
```

Comment:
Was changing `server` to `key_server` intentional? `key_server` was removed in puppetlabs-apt v4.0.0 ...